docs/operators_and_staking/operators.md

```diff
@@ -0,0 +1,289 @@
+
+Once the domain epoch is finished, the operator can produce bundles from the new epoch.
+
+Any **operator** can add more stake by using the same functionality.
+
+### Checking your operatorID
+
+There are two ways to check your operatorID: 
+
+1. You can use PolkadotJS **[Network Explorer](https://polkadot.js.org/apps/#/explorer)**.
+
+ ![Staking-7](/img/doc-imgs/operators-staking/Staking-7.png)
+2. Browse the **recent events** and you should see **domains.OperatorRegistered** event.
+
+ ![Staking-8](/img/doc-imgs/operators-staking/Staking-8.png)
+3. Click on the dropdown arrow to view the **domainID** and **operatorID**.
+
+---
+
+Alternatively, you can use [Subscan](https://subspace.subscan.io/) which provides a slightly better UI. 
+1. Navigate to **[Subspace Subscan](https://subspace.subscan.io/)** portal.
+2. Click on `Blockchain` -> `Extrinsics`.
+
+  ![Staking-9](/img/doc-imgs/operators-staking/Staking-9.png)
+
+3. Scroll to the bottom of the page to view all recent events, search for `register_operator` event.
+
+ ![Staking-10](/img/doc-imgs/operators-staking/Staking-10.png)
+
+4. Click on `Extrinsic ID` for the desired event.
+
+5. Scroll to `Parameters` and ensure that `signing_key` corresponds to your signing key.
+
+ ![Staking-11](/img/doc-imgs/operators-staking/Staking-11.png)
+
+6. Scroll to `Events` and click on dropdown arrow for `domains(OperatorRegistered)`.
+
+  ![Staking-12](/img/doc-imgs/operators-staking/Staking-12.png)
+
+7. Inspect and remember your `domain_id`. 
+
+
+### Start the domain operator node
+
+
+The domain operator node is running with an embedded consensus node, thus you need to specify the args for both the consensus node and the domain operator node:
+
+```bash
+subspace-node [consensus-chain-args] -- [domain-args]
+```
+
+
+Example:
+Start a node as operator on `gemini-3f` chain:
+
+<Tabs groupId="OS">
+<TabItem value="windows" label="🖼️ Windows" default>
+
+```powershell
+target/production/subspace-node `
+--chain gemini-3f `
+--rpc-external `
+--node-key 0000000000000000000000000000000000000000000000000000000000000001 `
+-- `
+--domain-id 3 `
+--chain gemini-3f `
+--operator `
+--keystore-path /tmp/keystore `
+--rpc-external
+```
+
+</TabItem>
+
+<TabItem value="macos" label="🍎 macOS">
+
+```bash
+target/production/subspace-node \
+--chain gemini-3f \
+--rpc-external \
+--node-key 0000000000000000000000000000000000000000000000000000000000000001 \
+-- \
+--domain-id 3 \
+--chain gemini-3f \
+--operator \
+--keystore-path /tmp/keystore \
+--rpc-external
+```
+
+</TabItem>
+
+<TabItem value="linux" label="🐧 Ubuntu">
+
+```bash
+target/production/subspace-node \
+--chain gemini-3f \
+--rpc-external \
+--node-key 0000000000000000000000000000000000000000000000000000000000000001 \
+-- \
+--domain-id 3 \
+--chain gemini-3f \
+--operator \
+--keystore-path /tmp/keystore \
+--rpc-external
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+The `node-key` in the example above can by any `hex` string of the same length.
```

Comment:
Are we still including that node-key can be any hex string? Should we prompt users to put in their own node-keys to avoid confusion

Reply:
Oh right, I don't think it needs to be set to the public key (hex) generated earlier, but I think it's still a good point to avoid confusion.